docs/maps/index.rst

```diff
@@ -125,6 +125,43 @@ Fermi-LAT PSF:
    m_hpx = Map.create(binsz=binsz, map_type='hpx', skydir=position, width=10.0,
                           axes=[energy_axis])
 
+Indexing and Slicing
+--------------------
```

Comment:
I think there needs to be an empty line after a section heading in RST for this to render correctly? (not 100% sure, didn't try)

Reply:
Done